tests/unit_tests/job_queue/test_lsf_driver.py

```diff
@@ -192,3 +196,22 @@ def test_run_mocked_lsf_queue():
     log = Path("bsub_log").read_text(encoding="utf-8")
     for i in range(10):
         assert f"'-J', 'poly_{i}'" in log
+
+
+async def test_that_driver_submit_triggers_statechange_accept(tmpdir, monkeypatch):
+    monkeypatch.chdir(tmpdir)
+    make_mock_bsub(Path("mocked_bsub"))
+    lsfdriver = LSFDriver([("BSUB_CMD", "./mocked_bsub")])
+
+    class MockedQueue:
+        _statechanges_to_publish = None
+
+    runarg = RunArg(
```

Comment:
I don't like the `RunArg` name to be honest (it's beyond this PR of-course).

Reply:
Me neither...

Reply:
But it is a valid point. We need to mock `RunArg` only because the `realization_state.py` picks `iens` from it. If we stop doing that, or provide `iens` in another way, we would not have to.